codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/json/service-2.json

```diff
@@ -11,7 +11,8 @@
     "signingName": "json-service",
     "signatureVersion": "v4",
     "uid": "json-service-2010-05-08",
-    "xmlNamespace": "https://json-service.amazonaws.com/doc/2010-05-08/"
+    "xmlNamespace": "https://json-service.amazonaws.com/doc/2010-05-08/",
+    "awsQueryCompatible": {}
```

Comment:
Can we add a separate file here to test this since this is specific to only one service now and this will fail to test when  "awsQueryCompatible": {} is not existent in c2j model